contracts/plugins/assets/curve/CurveStableMetapoolCollateral.sol

```diff
@@ -94,7 +94,6 @@ contract CurveStableMetapoolCollateral is CurveStableCollateral {
         // {UoA/tok} = {UoA} / {tok}
         low = aumLow.div(supply, FLOOR);
         high = aumHigh.div(supply, CEIL);
-        assert(low <= high); // not obviously true just by inspection
```

Comment:
I removed it because it's also checked here: https://github.com/reserve-protocol/protocol/blob/b5b4b641888178644c2ca269d4338ba15472785c/contracts/plugins/assets/Asset.sol#L116

but I'm realizing this could cause subtly different behavior: we should have the assert in both places actually. the only reason we should skip it in `tryPrice()` is if it's obviously always true, already. So I'll add it back, good catch